packages/create-rainbowkit/templates/next-app/pages/_app.tsx

```diff
@@ -24,19 +26,32 @@ const chains = [
   ...(process.env.NEXT_PUBLIC_ENABLE_TESTNETS === 'true' ? [sepolia] : []),
 ];
 
-const wagmiConfig = getDefaultConfig({
+const config = getDefaultConfig({
   appName: 'RainbowKit App',
   projectId: 'YOUR_PROJECT_ID',
   chains,
+  transports: {
+    [mainnet.id]: http(),
+    [polygon.id]: http(),
+    [optimism.id]: http(),
+    [arbitrum.id]: http(),
+    [base.id]: http(),
+    [zora.id]: http(),
+    [sepolia.id]: http(),
+  },
```

Comment:
In this case, the user shouldn't need to pass `transports`. We'd automatically infer and map to `http()` in `getDefaultConfig`